### ISSUE TITLE
CRAN update

### ISSUE THREAD

**Issue by lorenzwalthert** [dkahle/ggmap#212]
Hi

Thanks for this useful package. I saw that there are almost 2 years since the last CRAN release but there is some activity in the repo. Is there any intend for an update on CRAN?

**Comment by scottmmjackson** [comment]
Yes, we're hoping to improve geocoding before the next CRAN submission though.

**Comment by lorenzwalthert** [comment]
Cool. Thanks for letting me know.

**Comment by rawls238** [comment]
any update on when the next CRAN release will be?

**Comment by bamnet** [comment]
+1.  Access to Google Maps is going to start requiring an API Key any day now[1] and, AFAICT, the current version of this package on CRAN does not provide the `register_google()` function needed to configure an API key.

[1] https://twitter.com/GMapsPlatform/status/1047274648684187648

**Comment by dkahle** [comment]
Here's the patch for the time being:
```r
if(!requireNamespace("devtools")) install.packages("devtools")
devtools::install_github("dkahle/ggmap", ref = "tidyup")
```
Register the API key with `register_google()`.
